src/main/scala/cromwell/binding/Workflow.scala

```diff
@@ -28,6 +28,25 @@ case class Workflow(name: String, calls: Set[Call]) extends Scope {
     }
 
     /**
+      * Convert a fully qualified parameter name to a local parameter name.
+      *
+      * @param fqn Fully qualified parameter name
+      * @return local parameter name
+      *
+      * FIXME brittle
+      */
+    def fqnToLocal(fqn: String): String = fqn.substring(fqn.lastIndexOf(".") + 1)
+
+  /**
+   * All inputs for this workflow and their associated types, input names localized.
+   *
+   * @return Map of inputs to WdlValues with the input names localized.
+   */
+    def localizedInputs: Map[String, WdlType] = inputs.map { fqnInput =>
+      fqnToLocal(fqnInput._1) -> fqnInput._2
+    }
```

Comment:
I do really think we should figure out a way to not have these functions... perhaps we can talk about it?